models/activities/action.go

```diff
@@ -393,10 +393,10 @@ func (a *Action) GetCreate() time.Time {
 	return a.CreatedUnix.AsTime()
 }
 
-// GetIssueInfos returns a list of issues associated with
-// the action.
+// GetIssueInfos returns a list of associated information with the action.
 func (a *Action) GetIssueInfos() []string {
-	return strings.SplitN(a.Content, "|", 3)
+	// make sure it always returns 3 elements, because there are some access to the a[1] and a[2] without checking the length
+	return strings.SplitN(a.Content+"||", "|", 3)
```

Comment:
Thanks for the comment but that's wrong now:
```go
fmt.Printf("%#v", strings.SplitN("1|2|3"+"||", "|", 3))
// []string{"1", "2", "3||"}
```

Reply:
Already reverted with a new comment